layout-fetcher.js

```diff
@@ -29,16 +29,16 @@ module.exports = function(url, options) {
             return next();
         }
 
-        var requestOptions = {
-            url: url,
+        var requestOptions = { url: url };
 
-            headers: {
+        if (req.headers.cookie !== undefined) {
+             requestOptions.headers = {
                 // Pass cookies from the client through to the layout service
                 cookie: req.headers.cookie,
                 // Provide the remote IP address to the layout service
                 'X-Forwarded-For': req.connection.remoteAddress
```

Comment:
We will want to still pass through the `req.connection.remoteAddress` even when there are no cookies :)